Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst

```diff
@@ -70,20 +75,98 @@ Procedure Steps
 .. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
 .. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
 
+.. note::
 
+    Before/right after applying LOTO on the OSS, announce "LOTO on the OSS" on  *#summit-announce* and/or *#summit-simonyi*
 
-.. _Simonyi-LOTO-procedure-Troubleshooting:
 
-Troubleshooting
-===============
+Applying LOTO on OSS
+--------------------
+
+#.  Turn :guilabel:`OFF` the OSS in the EUI panel (indicated by the :gold:`circle` in the following Figure)
+
+    .. figure:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-1.png
+     :name: tma-loto-1
+
+#.  **Wait until the OSS is off**:  the :guilabel:`status` turns `IDLE` and :guilabel:`OSS` flag goes from green to gray. 
+    It takes around 5-10 minutes to complete turning off the OSS. 
+
+    .. figure:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-2.png
+     :name: tma-loto-2
+
+#.  Go to the OSS room from level 1 (first floor). Apply LOTOs on the OSS pump switches (highlighted by the :red:`box` in
+    the following Figure). Close valves (Position "—-") and put locks on the holes to apply three LOTOs on the safety 
+    switches of the OSS pumps.
+
+    .. figure:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-3.png
+     :name: tma-loto-3
+
+    .. image:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-4.png
+     :width: 30%
+    .. image:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-5.png
+     :width: 30%
+    .. image:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-6.png
+     :width: 30%
+
+.. note::
+
+    **LOTO box**
```

Comment:
Consider moving this LOTO box note before figures to line 100 or before first step (line 77) so it applies to Lock and Unlock procedures.